README.md

```diff
@@ -46,12 +46,18 @@ AND FURTHER DOWN IN `config/environment.js`:
 
 ```js
 if (environment === 'production') {
-  ENV.baseURL = '/';
+  ENV.rootURL = '/<name-of-git-repo>';
   ENV.locationType = 'hash';
-  ENV.apiHost = '<% replace with the URL to your deployed API %>'
+  ENV.apiHost = '<% replace with the URL to your deployed API %>';
 }
 ```
 
+1.  Now change the value of ENV.rootURL to be the name of your git repository; e.g. in the case of this repository it would be `ENV.rootURL = '/ember-deployment-guide'`
```

Comment:
So mine is working as '/<repo>'

I followed this convention in accordance with ga-wdi-boston/capstone-project/issues/330

And confirmed that detail in Rebekah's deployed application [here](https://github.com/rhjones/quiltr/blob/master/config/environment.js) @ga-wdi-boston/core does anyone else have time to try testing this as well? I can nuke pave and redeploy to confirm, but mine currently works like this.